### DISCUSSION TITLE
row level style :  mantineTableBodyRowProps  usage

### DISCUSSION THREAD

**Discussion by 24AMK10** [mantinedev/mantine#6980] [Q&A]
I want to highlight few rows according to specific condition and change bgColor and border of the row, how can I do that ?
My code is below but this doesn't work ...... 
` <MantineReactTable
          table={table}
          mantineTableBodyRowProps={({ row }) => ({
            sx: {
              backgroundColor: row.original.isHighlighted ? 'red' : 'transparent',
              border: row.original.isHighlighted ? '2px solid red' : 'none', 
              borderRadius: '4px', // Optional: round corners
              padding: '8px', // Add some padding
              transition: 'background-color 0.3s ease, border 0.3s ease', 
            },
          })}
          mantineTableBodyProps={{
            striped: true,
            sx: (theme, row)=> ({
              backgroundColor: 'red',
              minHeight: 'unset',
            }),
          }}
        />`

**Comment by rtivital** [comment]
To ask question about this project, use a dedicated repository for Mantine React Table, it is not maintained by the main team

**Reply by 24AMK10** [reply]
Thank you @rtivital ! 
I'll try to ask them, No worries.
